workflow/rocoto/gefs_tasks.py

```diff
@@ -387,7 +387,14 @@ def wavepostbndpntbll(self):
         deps = []
         atmos_hist_path = self._template_to_rocoto_cycstring(self._base["COM_ATMOS_HISTORY_TMPL"], {'MEMDIR': 'mem#member#'})
         # Is there any reason this is 180?
-        data = f'{atmos_hist_path}/{self.cdump}.t@Hz.atm.logf180.txt'
+        # The wavepostbndpntbll job runs on forecast hours up to 180 hours
+        last_fhr = Tasks._get_forecast_hours(self.cdump,
+                                             self._configs['wavepostbndpntbll'],
+                                             component='atmos')[-1]
+        if last_fhr > 180:
+            last_fhr = 180
+
+        data = f'{atmos_hist_path}/{self.cdump}.t@Hz.atm.logf{last_fhr:03d}.txt'
```

Comment:
This should be based on `FHMAX_WAV_IBP` from `config.wave`.

Reply:
Why is wavepostbndpntbll being added to GEFS? This is not a job GEFS runs.

Reply:
That's is my fault. Wasn't thinking when I added the wave post jobs to GEFS. I just copied what GFS had.

Reply:
I'll remove the boundary point jobs from GEFS later

Reply:
Thanks for the heads up @WalterKolczynski-NOAA , they're all turned off by default, but I was confused by this PR updating it (which I thought was adding it).

Reply:
Fixed.  Thanks!